mypyc/test-data/run-classes.test

```diff
@@ -2503,3 +2503,35 @@ class C:
 def test_final_attribute() -> None:
     assert C.A == -1
     assert C.a == [-1]
+
+[case testClassWithFinalTypingAttribute]
+from typing import final
+
+@final
+class C:
+    def a(self) -> int:
```

Comment:
Test accessing an attribute and a property of a final class?

Reply:
Done, now testClassWithFinalTypingAttributeInherited try accessing property and attribute on parent and child class